presto-main/src/main/java/com/facebook/presto/server/InternalCommunicationConfig.java

```diff
@@ -84,13 +85,25 @@ public String getTrustStorePath()
         return trustStorePath;
     }
 
+    public String getTrustStorePassword()
```

Comment:
Can you move this getter to be next to the corresponding setter?

Reply:
Sure! Just updated my PR accordingly.